tasks/nsp.js

```diff
@@ -36,8 +41,13 @@ module.exports = function (grunt) {
 
     if (grunt.option('output')) {
       formatter = Nsp.formatters[grunt.option('output')];
-    }
 
+      if (formatters.indexOf(config.output) !== -1) {
```

Comment:
Not sure if ESLint complains (if we're running it against this repo), but it almost looks like `formatters` is out of scope here since it was declared within a `if(){...}` block above (but not really due to variable hoisting).